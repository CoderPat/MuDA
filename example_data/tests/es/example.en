Louis XIV had a lot of people working for him.
They made his silly outfits, like this.