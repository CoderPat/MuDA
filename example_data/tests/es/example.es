Luis XIV tenía un montón de gente trabajando para él.
Ellos hacían sus trajes tontos, como éste.