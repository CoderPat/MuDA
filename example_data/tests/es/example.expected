0 0 0 0 0 0 0 0 0 0 0
4 3 0 0 0 0 0 4 0